packages/react-composites/src/composites/CallWithChatComposite/CallWithChatPane.tsx

```diff
@@ -0,0 +1,153 @@
+import {
+  paneBodyContainer,
+  scrollableContainer,
+  scrollableContainerContents
+} from '../common/styles/ParticipantContainer.styles';
+import { BasicHeader } from './BasicHeader';
+import { useCallWithChatCompositeStrings } from './hooks/useCallWithChatCompositeStrings';
+import { LocalAndRemotePIPInModal, LocalAndRemotePIPInModalStyles } from './LocalAndRemotePIPInModal';
+import { PeoplePaneContent } from './PeoplePaneContent';
+import { drawerContainerStyles } from './styles/CallWithChatCompositeStyles';
+import { TabHeader } from './TabHeader';
+/* @conditional-compile-remove(file-sharing) */
+import { FileSharingOptions } from '../ChatComposite';
+
+/**
+ * Pane that is used to store chat and people for CallWithChat composite
+ * @private
+ */
+export const CallWithChatPane = (props: {
+  chatCompositeProps: Partial<ChatCompositeProps>;
+  callAdapter: CallAdapter;
+  chatAdapter: ChatAdapter;
+  onClose: () => void;
+  onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
+  onFetchParticipantMenuItems?: ParticipantMenuItemsCallback;
+  onChatButtonClicked: () => void;
+  onPeopleButtonClicked: () => void;
+  modalLayerHostId: string;
+  activePane: 'chat' | 'people' | 'none';
+  mobileView?: boolean;
+  inviteLink?: string;
+  /* @conditional-compile-remove(file-sharing) */
+  fileSharing?: FileSharingOptions;
+}): JSX.Element => {
+  const [drawerMenuItems, setDrawerMenuItems] = useState<_DrawerMenuItemProps[]>([]);
+
+  const hidden = props.activePane === 'none';
+  const paneStyles = hidden ? hiddenStyles : props.mobileView ? availableSpaceStyles : sidePaneStyles;
+
+  const callWithChatStrings = useCallWithChatCompositeStrings();
+  const theme = useTheme();
+
+  const header =
+    props.activePane === 'none' ? null : props.mobileView ? (
+      <TabHeader {...props} activeTab={props.activePane} />
+    ) : (
+      <BasicHeader
+        {...props}
+        headingText={
+          props.activePane === 'chat' ? callWithChatStrings.chatPaneTitle : callWithChatStrings.peoplePaneTitle
+        }
+      />
+    );
+
+  const pipStyles: LocalAndRemotePIPInModalStyles = useMemo(
+    () => ({
+      modal: {
+        main: {
+          borderRadius: theme.effects.roundedCorner4,
+          boxShadow: theme.effects.elevation8,
+          ...(theme.rtl ? { left: '1rem' } : { right: '1rem' })
+        }
+      }
+    }),
+    [theme.effects.roundedCorner4, theme.effects.elevation8, theme.rtl]
+  );
+
+  return (
+    <Stack
+      verticalFill
+      grow
+      styles={paneStyles}
+      data-ui-id={
+        props.activePane === 'chat' ? 'call-with-chat-composite-chat-pane' : 'call-with-chat-composite-people-pane'
+      }
+    >
+      {header}
+      <Stack.Item verticalFill grow styles={paneBodyContainer}>
+        <Stack horizontal styles={scrollableContainer}>
+          <Stack.Item verticalFill styles={scrollableContainerContents}>
+            <Stack styles={props.activePane === 'chat' ? availableSpaceStyles : hiddenStyles}>
```

Comment:
nit, the body of this is v. long and the nesting makes it hard to visualize, might be worth splitting components out and having something like:
```
<Stack>
  {header}
  <Stack.Item>
    <Stack>
      <Stack.Item>
        {chatPane}
        {peoplePane}
     </Stack.Item>
    </Stack>
   </Stack.Item>
   {modalPiPiP}
</Stack> 
```

So it's easier to visualize how these components live relative to each other